src/main/java/com/cst438/controller/AssignmentController.java

```diff
@@ -14,27 +14,44 @@
 import org.springframework.web.server.ResponseStatusException;
 
 
+import java.sql.Date;
 import java.util.ArrayList;
 import java.util.List;
 
 @RestController
 @CrossOrigin(origins = "http://localhost:3000")
 public class AssignmentController {
 
+    @Autowired
+    private AssignmentRepository assignmentRepository;
+
+    @Autowired
+    private SectionRepository sectionRepository;
+
+    @Autowired
+    private GradeRepository gradeRepository;
+
+    @Autowired
+    private EnrollmentRepository enrollmentRepository;
 
     // instructor lists assignments for a section.  Assignments ordered by due date.
     // logged in user must be the instructor for the section
     @GetMapping("/sections/{secNo}/assignments")
     public List<AssignmentDTO> getAssignments(
-            @PathVariable("secNo") int secNo) {
-
-        // TODO remove the following line when done
-		
-		// hint: use the assignment repository method 
-		//  findBySectionNoOrderByDueDate to return 
-		//  a list of assignments
-
-        return null;
+            @PathVariable("secNo") int secNo) {Section section = sectionRepository.findBySectionNum(secNo);
```

Comment:
This should be on a new line

Reply:
Should also throw an error if the section can't be found.

Reply:
@PathVariable("secNo") int secNo) {Section section = sectionRepository.findBySectionNum(secNo).orElseThrow(()
            -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Section not found"));